app/views/layouts/base.html.erb

```diff
@@ -41,7 +38,7 @@
 
   <body class="govuk-template__body <%= yield :body_class %>">
     <script nonce="<%= request.content_security_policy_nonce %>">
-      document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+      document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');
```

Comment:
💡FYI: As an alternative to this approach, JS Elements have a `classList` method which allows us to add classes easily using `classList.add("class-to-be-added")`
See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) for more detail

Reply:
Thanks @dcyoung-dev! I am going to close this PR as it's been superseded by [this one](https://github.com/DFE-Digital/apply-for-teacher-training/pull/9092/files)

It will just cause confusion now.